lib/src/modules/task.cc

```diff
@@ -510,12 +510,20 @@ void Task::processOutputAndUpdateMetadata(ActionResponse& response)
     std::string &output = response.output.std_out;
 
     if (isValidUTF8(output)) {
-        lth_jc::JsonContainer stdout_result;
-        stdout_result.set("output", output);
+        // Use a temporary object to create a JSON string. The JsonContainer API doesn't
+        // provide a way to set the root to a string unless that string is already quoted.
+        lth_jc::JsonContainer result;
```

Comment:
This avoids the mistake we made with external modules, by returning all the relevant results in the non-blocking response.